clouddriver-ecs/src/integration/java/com/netflix/spinnaker/clouddriver/ecs/test/LoadBalancersSpec.java

```diff
@@ -0,0 +1,254 @@
+
+    String testNamespaceForService = SERVICES.ns;
+    String serviceKey =
+        com.netflix.spinnaker.clouddriver.ecs.cache.Keys.getServiceKey(
+            ECS_ACCOUNT_NAME, "us-west-2", "TestAgentService");
+
+    Map<String, Object> serviceAttributes = new HashMap<>();
+    serviceAttributes.put("account", ECS_ACCOUNT_NAME);
+    serviceAttributes.put("region", TEST_REGION);
+    serviceAttributes.put("applicationName", "TestAgentService");
+    serviceAttributes.put("loadBalancers", Arrays.asList(loadBalancer));
+    serviceAttributes.put("serviceName", "testService");
+    serviceAttributes.put("serviceArn", "service/testServiceArn");
+    serviceAttributes.put("clusterName", "ecsTestCluster");
+    serviceAttributes.put("clusterArn", "cluster/testClusterArn");
+    serviceAttributes.put("roleArn", "role/testRoleArn");
+    serviceAttributes.put("taskDefinition", "testTaskDefinition");
+    serviceAttributes.put("desiredCount", 1);
+    serviceAttributes.put("maximumPercent", 10);
+    serviceAttributes.put("minimumHealthyPercent", 10);
+    serviceAttributes.put("subnets", Arrays.asList("testSubnet"));
+    serviceAttributes.put("securityGroups", Arrays.asList("test-security"));
+    serviceAttributes.put("createdAt", createdAtLong);
+
+    DefaultCacheResult testResultForService =
+        buildCacheResult(serviceAttributes, testNamespaceForService, serviceKey);
+    ecsCache.addCacheResult(
+        "TestAgentService",
+        Collections.singletonList(testNamespaceForService),
+        testResultForService);
+
+    String testNamespaceForTG = TARGET_GROUPS.ns;
+
+    String targetGroupKey =
+        com.netflix.spinnaker.clouddriver.aws.data.Keys.getTargetGroupKey(
+            "spinnaker-ecs-demo-tg", "aws-account", TEST_REGION, "", "vpc-123");
+
+    Map<String, Object> targetGroupAttributes = new HashMap<>();
+    targetGroupAttributes.put("loadBalancerNames", Arrays.asList("ecsLB"));
+    targetGroupAttributes.put("targetGroupName", "spinnaker-ecs-demo-tg");
+    targetGroupAttributes.put(
+        "targetGroupArn",
+        "aws:targetGroups:aws-account:us-west-2:spinnaker-ecs-demo-tg:ip:vpc-07daae48bf98a8fd8");
+
+    DefaultCacheResult testResultForTargetGroup =
+        buildCacheResultForLB(
+            targetGroupAttributes, testNamespaceForTG, targetGroupKey, relationships);
+    ecsCache.addCacheResult(
+        "TestAgentTG", Collections.singletonList(testNamespaceForTG), testResultForTargetGroup);
+
+    when(mockEcsAccountMapper.fromAwsAccountNameToEcsAccountName("*")).thenReturn("ecs-account");
+    when(mockEcsAccountMapper.fromEcsAccountNameToAwsAccountName(ECS_ACCOUNT_NAME))
+        .thenReturn("aws-account");
+
+    String url = getTestUrl("/applications/TestAgentService/loadBalancers");
+
+    // when
+    Response response = get(url).then().contentType(ContentType.JSON).extract().response();
+    log.info("LOAD BALANCERS response:" + response.asString());
+
+    String responseStr = response.asString();
```

Comment:
Out of curiosity, it is possible to deserialize response into the actual object (so that you can check that the correct values are defined for the right fields)?

Reply:
@paragbhingre where did we land on this change/suggestion?

Reply:
@allisaurus sorry for the delay. But deserializing response into an object for the **getLoadBalancersTest** is not possible due changing parameter names in the response as we discussed previously. For **getLoadBalancersForApplicationTest** I was able to deserialize into an object. 
I guess we should go ahead with the string response comparison for getLoadBalancersTest as we are not able to deserialize it properly. What's your call on this one?

Reply:
fine with not blocking on deserializing `getLoadBalancersTest` response for now so long as we are still asserting on the presence of expected data.